.tool-versions

```diff
@@ -2,3 +2,4 @@ terraform 1.5.7
 terraform-docs 0.17.0
 tflint 0.50.3
 pre-commit 3.6.2
+python 3.12.1
```

Comment:
Remove python from here, we don't need it.

Reply:
ok